modules/dnn/src/layers/convolution_layer.cpp

```diff
@@ -488,11 +505,11 @@ class ConvolutionLayerImpl CV_FINAL : public BaseConvolutionLayerImpl
 #if INF_ENGINE_VER_MAJOR_GE(INF_ENGINE_RELEASE_2018R5)
         InferenceEngine::Builder::ConvolutionLayer ieLayer(name);
 
-        ieLayer.setKernel({(size_t)kernel.height, (size_t)kernel.width});
-        ieLayer.setStrides({(size_t)stride.height, (size_t)stride.width});
-        ieLayer.setDilation({(size_t)dilation.height, (size_t)dilation.width});
-        ieLayer.setPaddingsBegin({(size_t)pad.height, (size_t)pad.width});
-        ieLayer.setPaddingsEnd({(size_t)pad.height, (size_t)pad.width});
+        ieLayer.setKernel(std::vector<size_t>(kernel_size.begin(), kernel_size.end()));
+        ieLayer.setStrides(std::vector<size_t>(strides.begin(), strides.end()));
+        ieLayer.setDilation(std::vector<size_t>(dilations.begin(), dilations.end()));
+        ieLayer.setPaddingsBegin(std::vector<size_t>(pads.begin(), pads.begin() + pads.size() / 2));
+        ieLayer.setPaddingsEnd(std::vector<size_t>(pads.begin() + pads.size() / 2, pads.end()));
```

Comment:
Can we just use `std::vector<size_t>` data type for all the `kernel_size`, `pads`, `strides` and `dilations`?